contents/resources/riccardo-cultural-innovation/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Cultural Innovation
+date: 2020-07-01
+authors:
+  - riccardo-pozzo
+categories:
+  - desir
+tags:
+  - research-infrastructures
+abstract:
+  This video features a keynote speech given by Riccardo Pozzo, in his capacity
+  as Chair of the DARIAH Scientific Board, during the final event for the DESIR
+  project, held in Zagreb in November 2019.  In it, he discusses co-creation and
+  cultural innovation, and how research infrastructures play a key role in this.
+
+type: video
+remoteUrl: https://www.youtube.com/watch?v=8M-xhOwydgE
+remotePublicationDate: 2020-05-08
+---
+
+Riccardo Pozzo is chair of the History of Philosophy at University of Rome Tor
+Vergata and chair of the DARIAH Scientific Board. The focus of his keynote is on
+the introduction of the notion of cultural innovation, which requires adapting
+the process of co-creation. The dimension of cultural innovation is contrasted
+against other forms of innovation, attempting to point out processes and
+outcomes of cultural innovation while showing their operationalisation in some
+empirical case studies.
+
+The keynote was recorded at the DESIR Final Event, in November 2019, in Zagreb,
+Croatia. The event was organised in the framework of the Horizon 2020 funded
+project DESIR (DARIAH ERIC Sustainability Refined).
+
+<VideoCard
+  title="DH in Practice"
+  subtitle="Cultural Innovation - Riccardo Pozzo at the DESIR Final Event 2019"
+  url="https://www.youtube.com/watch?v=8M-xhOwydgE"
```

Comment:
Video cards don't take an `url` attribute, but in `id` attribute. If you put it like this, you will see that the video won't play. That's why it's always important to preview things before submitting them.

Reply:
and the `id` attribute takes just the id, i.e. in this case "8M-xhOwydgE"